### DISCUSSION TITLE
zh-hant/chapter_hello_algo/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1229] [Announcements]
# zh-hant/chapter_hello_algo/

動畫圖解、一鍵執行的資料結構與演算法教程

https://www.hello-algo.com/zh-hant/chapter_hello_algo/

**Comment by lumao1106** [comment]
哇嗚，突然看到繁體中文了，好用心，非常感謝(⁎⁍̴̛ᴗ⁍̴̛⁎)

**Reply by krahets** [reply]
久等了！如果發現問題，歡迎在 [#1171](https://github.com/krahets/hello-algo/issues/1171) 勘誤

**Comment by brian-huanggg** [comment]
很喜歡作者的一句話：

> 兩眼一抹黑地刷題似乎是最受歡迎的方法，簡單、直接且有效。然而刷題就如同玩“掃雷”遊戲，自學能力強的人能夠順利將地雷逐個排掉，而基礎不足的人很可能被炸得滿頭是包，並在挫折中步步退縮

真的身為一個過來人...刷題看是有效但往往也可能是最沒有效率的方法🥲
